Source/debug.cpp

```diff
@@ -184,22 +197,23 @@ std::string DebugCmdLoadMap(const string_view parameter)
 	auto level = atoi(parameter.data());
 	if (level < 1)
 		return fmt::format("Map id must be 1 or higher", level);
-	if (setlevel && myPlayer.plrlevel == level)
-		return fmt::format("I did nothing but fulfilled your wish. You are already at level {}.", level);
+	if (setlevel && setlvlnum == level)
+		return fmt::format("I did nothing but fulfilled your wish. You are already at mapid {}.", level);
 
 	for (auto &quest : Quests) {
-		if (level != quest._qslvl)
+		if (level != quest._qslvl || quest._qactive == quest_state::QUEST_NOTAVAIL)

```

Comment:
Won't this result in it reporting maps for inactive quests to be reported as unknown?

I's the a good reason we should not allow the player to jump to a level from an inactive quesets when debugging?

Reply:
inactive quest maps have no exit so you end up at 0,0 if you leave them manually

Reply:
That isn't really an issue when debugging, just `changelevel 0` and your good.

Reply:
Personally I would disable the map if the quest is not active.
But I don't want to argue too much about a debug command, so I removed the commit. 😉